Components/Sources/Components/Components/Onboarding/WKOnboardingViewController.swift

```diff
@@ -4,10 +4,17 @@ import SwiftUI
 public protocol WKOnboardingViewDelegate: AnyObject {
     func didClickPrimaryButton()
     func didClickSecondaryButton()
+    func willSwipeToDismiss()
```

Comment:
One recommendation with these delegate methods would be to prefix them with the delegating type, so in this case something like `onboardingViewWillSwipeToDismiss` or something (akin to how table view or collection view delegate methods might be be named). As we see later on with the `ViewControllerRouter` and `WKImageRecommendationsViewController` conformance, the general `willSwipeToDismiss` or `didClickPrimaryButton` etc. naming makes it confusing as to the function's intent.

Reply:
Agreed - Done!